cookbooks/aws-parallelcluster-slurm/recipes/config/login_nodes_daemon_service.rb

```diff
@@ -0,0 +1,54 @@
+require 'yaml'
+config = YAML.safe_load(File.read(node['cluster']['cluster_config_path']))
+GRACETIME_PERIOD = config["LoginNodes"]["Pools"][0]["GRACETIME_PERIOD"]
```

Comment:
I don't think so, because I saw the cluster-config.yaml file in shared directory in instance. It's like:
```
LoginNodes:
  Pools:
  - Count: 2
    GracetimePeriod: 6
    Iam:
      AdditionalIamPolicies: []
      InstanceProfile: null
      InstanceRole: null
    Image: null
    InstanceType: t2.micro
    Name: test
    Networking:
      AdditionalSecurityGroups: null
      Proxy: null
      SecurityGroups: null
      SubnetIds:
      - subnet-0c45c7238162199db
    Ssh:
      KeyName: xuanqi-us-east-2
```
So it will show the entire config yaml and get the default value of 60 minutes.